### DISCUSSION TITLE
ERR unable to proxy Istiod pods. Make sure your Kubernetes API server has access to the Istio control plane through 8080 port

### DISCUSSION THREAD

**Discussion by ciherrera** [kiali/kiali#4513] [Q&A]
desplegué istio 1.11 con kiali 1.38.1  en un GKE y estoy teniendo este error , he habilitado los puertos indicados en la documentación  con  gcloud compute firewall-rules update <firewall-rule-name> --allow <previous-ports>,tcp:8080,tcp:15000   pero aún asi no funciona alguna idea ?
![image](https://user-images.githubusercontent.com/49250191/142611664-306566bd-2ef0-490c-93b9-02ea7cff125f.png)

**Comment by xeviknal** [comment]
There is the error also in the discussions: https://github.com/kiali/kiali/discussions/4239 Does this helps?

Also, please do you mind adding your kubernetes version and flavor? Are you using kubernetes in any cloud provider (e.g. gcp, aws, ibm)?
Which version of kiali are you using? How did you install it? helm? operator? istio plugin?

Thank you

**Comment by ciherrera** [comment]
k8s 2.17,GCP, KIALI 1.42, installing like that

function install() {

      yes | istioctl install -f ./istio-1.11/istio-operator.yaml || sleep 30

      kubectl apply -f ./istio-1.11/istio-envoy-filter-headers.yaml

      ## Instalo los addons
      install_addons
    }

    function remove() {
      echo "Desinstalando"
      delete_addons

      yes | istioctl x uninstall --purge
    }

    
    function install_addons() {
      echo "Install addons...."
      kubectl apply -f ./addons/istio-destination-rule-mesh.yaml
      kubectl apply -f ./addons/general-gateways.yaml -n $API_NAMESPACE
      kubectl apply -f ./addons/grafana.yaml
      kubectl apply -f ./addons/vs/vs-grafana.yaml
      kubectl apply -f ./addons/jaeger.yaml
      kubectl apply -f ./addons/vs/vs-kiali.yaml
      kubectl apply -f ./addons/prometheus.yaml      
      kubectl apply -f ./addons/kiali.yaml -n istio-system
      
      kubectl label namespace $API_NAMESPACE istio-injection=enabled || true

      ## Roles para kiali
      kubectl -n istio-system create serviceaccount kiali-admin || true
      kubectl create clusterrolebinding kiali-admin-binding --clusterrole=kiali --serviceaccount=istio-system:kiali-admin || true
    } 

    function delete_crd() {
      kubectl delete crd clusterrbacconfigs.rbac.istio.io  || true
      kubectl delete crd rbacconfigs.rbac.istio.io || true
      kubectl delete crd servicerolebindings.rbac.istio.io || true 
      kubectl delete crd serviceroles.rbac.istio.io || true
    } 

    function delete_addons() {
      echo "Eliminando los Addons..."
      kubectl delete all -l app=grafana -n istio-system || true
      kubectl delete all -l app=prometheus -n istio-system || true
      kubectl delete all -l app=kiali -n istio-system || true
      kubectl delete all -l app=jaeger -n istio-system || true
    }

    function upgrade() {
      ## Elimino los addons
      delete_addons
      
      ## Elimino los rbac incompatibles entre version
      delete_crd
      
      kubectl apply -f ./istio-1.11/istio-envoy-filter-headers.yaml

      echo "Upgrade Istio version 1.11.3"
      yes | istioctl upgrade -f ./istio-1.11/istio-operator.yaml || sleep 30
